.drone.yml

```diff
@@ -1,168 +1,133 @@
 ---
 kind: pipeline
 type: kubernetes
-name: lint
-
+name: build
 trigger:
   event:
     - push
+  branch:
+    exclude:
+      - main
 
 steps:
-- name: lint bash
-  image: koalaman/shellcheck-alpine
-  commands:
-    - shellcheck *.sh */*.sh */*/*.sh
-  depends_on:
-    - clone
+  - name: build & push
+    image: plugins/docker
+    settings:
+      registry: quay.io
+      repo: quay.io/ukhomeofficedigital/hocs-toolbox
+      tags:
+        - ${DRONE_COMMIT_SHA}
+    environment:
+      DOCKER_PASSWORD:
+        from_secret: QUAY_ROBOT_TOKEN
+      DOCKER_USERNAME: ukhomeofficedigital+hocs_quay_robot
+    depends_on:
+      - clone
 
 ---
 kind: pipeline
 type: kubernetes
-name: build
+name: deploy
+trigger:
+  event:
+    - promote
 
-platform:
-  os: linux
-  arch: amd64
+steps:
+  - name: fetch and checkout
+    image: alpine/git
+    commands:
+      - git fetch --tags
+      - git checkout $${VERSION}
 
+  - name: deploy to prod
```

Comment:
I'm not sure there's much value in manually gating this particular repo's journey to production. Maybe we could make it deploy to prod automatically whenever it's a non-major version, or at least deploy to both prods whenever we do get round to manually deploying it?